content/300-guides/200-deployment/110-deployment-guides/300-deploying-to-vercel.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 title: 'Deploying to Vercel'
 metaTitle: 'How to deploy a Prisma app to Vercel'
-metaDescription: 'Learn how to deploy a Next.js app with a serverless REST API using Prisma Client to Vercel.'
+metaDescription: 'Learn how to use Prisma Client to deploy a Next.js app with a serverless REST API to Vercel.'
```

Comment:
This change sounds incorrect.

Reply:
Good spot. I've corrected this in the following PR:

[https://github.com/prisma/docs/pull/3572](https://github.com/prisma/docs/pull/3572)